tensorflow/python/eager/backprop_test.py

```diff
@@ -1858,6 +1858,10 @@ def _func(x):
                           rtol=1e-3)
 
   def test_grad_jacobian_conv(self):
+    import platform
+    if platform.system() == 'Darwin':
```

Comment:
Why does this test fail with the change?  Is the system not "Darwin" when running on CPU in macos without metal?